.github/workflows/cli-test.yml

```diff
@@ -109,3 +109,40 @@ jobs:
         if: matrix.os == 'windows-latest'
         run: |
           npm run test:cli-integration
+
+  report-failure:
+    name: Report failure on the default branch
+    runs-on: ubuntu-latest
+    needs: [cli-test]
+    if: failure() && github.ref == 'refs/heads/main'
+    permissions:
+      issues: write
+    steps:
+      - name: Create GitHub issue
+        run: |
+          # Set -eu so that we fail if the gh command fails.
+          set -eu
+
+          # Try to find an existing open issue if there is one
+          ISSUE="$(gh issue list --repo "$GITHUB_REPOSITORY" --label "cli-test-failure" --state "open" --limit 1 --json number -q '.[0].number')"
+
+          if [[ -n "$ISSUE" ]]; then
+            echo "Found open issue number $ISSUE ($GITHUB_SERVER_URL/$GITHUB_REPOSITORY/issues/$ISSUE)"
+          else
+            echo "Did not find an open tracking issue. Creating one."
+
+            ISSUE_BODY="issue-body.md"
+            printf "CLI tests have failed on the default branch.\n\n@github/code-scanning-secexp-reviewers" > "$ISSUE_BODY"
```

Comment:
Maybe this isn't relevant, but I was wondering who the issue author would be. Is it `github-actions` bot?

If so, I'm not sure that we'll get the team notification for `@github/code-scanning-secexp-reviewers` 🤔 (Since the "mentioner" would need to be part of the GitHub org to mention the team) 

We can try it out though and see what it looks like!

Reply:
Interesting point. I believe yes it'll be the `github-actions` bot. I wasn't aware that might block it from notifying us.

I suppose maybe the team mention might be unnecessary if anyone who wants these notifications subscribes to the label.